user_guide/intro_log_in_and_edit_profile.rst

```diff
@@ -0,0 +1,554 @@
+    existing one, the old key is discarded and cannot be used to access OroCRM.
+    
+    
+Download the Outlook Add-in
+"""""""""""""""""""""""""""
+
+OroCRM Enterprise Edition users will also find a link to the latest version of OroCRM add-in for Outlook. 
+With this add-in you can synchronize contacts, tasks, and calendar events between OroCRM and your Outlook.  
+You can associate emails to accounts, contacts, opportunities, and cases and create leads, opportunities and cases 
+in OroCRM based on emails from the Outlook side bar.
+
+|
+
+.. image:: ./img/intro/user_outlook.png
+
+|
+
+Click the link to start the download. The file will be saved subject to your browser settings. 
+
+To learn more about the add-in, please refer to the :ref:`Integration with Outlook guide <user-guide-synch-outlook>`.
+
+
+Get to the Edit Page of Your User Profile
+"""""""""""""""""""""""""""""""""""""""""    
+Click the :guilabel:`Edit` button in the top left corner of the page to change and add the profile details, as described 
+in the section below.
+
+|
+
+.. image:: ./img/intro/user_edit.png
+    
+|
+    
+.. _user-guide-getting-started-profile:
+    
+Edit Your User Profile
+-----------------------
+
+Once you have clicked the Edit, the Edit form of your user profile will be displayed. Here you can update your 
+credentials, change your password, and synchronize OroCRM with an existing IMAP mailbox.
+
+Manage Your User Details
+^^^^^^^^^^^^^^^^^^^^^^^^
+|
+
+.. image:: ./img/intro/user_edit_general.png
+
+|
+
+At the top of the *"General"* section, you can view, define and change the basic details such as:
+
+- **Username**: An alias used for login and identification. It should be at least three characters long and cannot 
+  start with a space.
+
+- **Names**: Your personal name. It can be comprised of the first, middle, and last name, a prefix and a suffix. The 
+  first and the last name are mandatory. You can use any number of characters of any alphabet in the name.
+
+- **Birthday**: Select the date in the calendar. 
+
+- **Avatar**: A picture associated with you in the system. To set an avatar, click the :guilabel:`Choose file` button 
+  and select your avatar file from your computer. Avoid using large files; 100×100 pixels is recommended. After you have 
+  saved the user settings the file will be checked for consistency with your organization’s rules.  Please contact your 
+  OroCRM administrator for details. The image will be displayed along with your name in the selectors and will help 
+  other users to recognize you quickly.
+
+  |
+  
+  |EditAvatar|
+
+  |
+  
+Any custom details permitted for editing will be available in the *"Additional"* section. In the example above, this is 
+the *"Title"* field.
+
+.. note::
+
+    Although the *"Status"* fields is displayed on the Edit user page, you cannot change it, as it is impossible to 
+    deactivate a profile that you are using.
```

Comment:
Edit: Although the *"Status"* field is displayed on the Edit user page, you cannot change it, since it is impossible to deactivate a profile that you are currently using.